db/model/room.js

```diff
@@ -61,6 +61,14 @@ module.exports = function room(seq, dataTypes) {
       allowNull: true,
       comment: 'Bot names to be used in rooms',
     },
+    origin: {
+      type: dataTypes.STRING,
```

Comment:
free text here is recipe for chaos! what are the values you actually expect to see here? would it be possible to make it an enum with "Other" as one of the options? then add options as needed down the line? it's not going to be infinite! i'm sure you can come up with a half dozen values or fewer

Reply:
@iamigo good point! changed to enum and added required options